docs(animations): improve the aio animations introduction (#42885)
angular/angular: aio/content/guide/animations.md

Use Angular's `state()` function to define different states to call at the end of each transition. This function takes two arguments: a unique name like `open` or `closed` and a `style()` function.

> Use the `style()` function to define a set of styles to associate with a given state name. You must write style attributes that contain dashes in [*camelCase*](guide/glossary#case-conventions), such as `'backgroundColor'` or denoted as strings, such as `'background-color'`.

Author: ah ok :slightly_smiling_face::+1: 

But I think something in your suggestion is missing... :thinking: 
`You must [*camelCase*](guide/glossary#case-conventions) for style attributes...`
did you missed something like "use"?
`You must use [*camelCase*](guide/glossary#case-conventions) for style attributes...`
or maybe the "for" is to be removed? (using camelCase as a verb)?
`You must [*camelCase*](guide/glossary#case-conventions) style attributes...`

> Use the `style()` function to define a set of styles to associate with a given state name. You must write style attributes that contain dashes in [*camelCase*](guide/glossary#case-conventions), such as `'backgroundColor'` or denoted as strings, such as `'background-color'`.

Reviewer: Sorry, I meant _"You must **use** camelCase for ..."_ 😇

> Use the `style()` function to define a set of styles to associate with a given state name. You must write style attributes that contain dashes in [*camelCase*](guide/glossary#case-conventions), such as `'backgroundColor'` or denoted as strings, such as `'background-color'`.

Author: done :smile:

> Use the `style()` function to define a set of styles to associate with a given state name. Note that the style attributes containing dashed must either be written in [*camelCase*](guide/glossary#case-conventions) (e.g. `'backgroundColor'`) or denoted as strings (e.g. `'background-color'`).

Reviewer 2: ```suggestion
Use the `style()` function to define a set of styles to associate with a given state name. You must write style attributes that contain dashes in [*camelCase*](guide/glossary#case-conventions), such as `'backgroundColor'` or denoted as strings, such as `'background-color'`.
```

</tr>

<tr>

> <td><code><a href="api/animations/state" class="code-anchor">state</a>()</code></td>

Author: all standard method references in the docs don't have the brackets included in the link
